frontend/src/components/HostMetricsDetail/index.tsx

```diff
@@ -0,0 +1,154 @@
+								<Typography.Text type="secondary">STATUS</Typography.Text>
+								<Typography.Text type="secondary">OPERATING SYSTEM</Typography.Text>
+								<Typography.Text type="secondary">CPU USAGE</Typography.Text>
+								<Typography.Text type="secondary">MEMORY USAGE</Typography.Text>
+							</div>
+							<div className="values-row">
+								<Tag className={`status-tag ${host.active ? 'active' : 'inactive'}`}>
+									{host.active ? 'Active' : 'Inactive'}
+								</Tag>
+								<Tag>{host.os}</Tag>
+								<div className="progress-wrapper">
+									<Progress
+										percent={Math.round(host.cpu * 100)}
+										size="small"
+										strokeColor={
+											host.cpu > 0.8 ? 'var(--error-500)' : 'var(--success-500)'
+										}
+									/>
+								</div>
+								<div className="progress-wrapper">
+									<Progress
+										percent={Math.round(host.memory * 100)}
+										size="small"
+										strokeColor={
+											host.memory > 0.8 ? 'var(--error-500)' : 'var(--success-500)'
+										}
+									/>
+								</div>
+							</div>
+						</div>
+					</div>
+
+					<Radio.Group
+						className="views-tabs"
+						onChange={handleModeChange}
+						value={selectedView}
+					>
+						<Radio.Button
+							className={
+								// eslint-disable-next-line sonarjs/no-duplicate-string
+								selectedView === VIEW_TYPES.CONTAINERS ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.CONTAINERS}
+						>
+							<div className="view-title">
+								<Package2 size={14} />
+								Containers
+							</div>
+						</Radio.Button>
+						<Radio.Button
+							className={
+								selectedView === VIEW_TYPES.PROCESSES ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.PROCESSES}
+						>
+							<div className="view-title">
+								<ChevronsLeftRight size={14} />
+								Processes
+							</div>
+						</Radio.Button>
+						<Radio.Button
+							className={
+								selectedView === VIEW_TYPES.METRICS ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.METRICS}
+						>
+							<div className="view-title">
+								<BarChart2 size={14} />
+								Metrics
+							</div>
+						</Radio.Button>
+						<Radio.Button
+							className={
+								selectedView === VIEW_TYPES.LOGS ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.LOGS}
+						>
+							<div className="view-title">
+								<ScrollText size={14} />
+								Logs
+							</div>
+						</Radio.Button>
+						<Radio.Button
+							className={
+								selectedView === VIEW_TYPES.TRACES ? 'selected_view tab' : 'tab'
+							}
+							value={VIEW_TYPES.TRACES}
+						>
+							<div className="view-title">
+								<DraftingCompass size={14} />
+								Traces
+							</div>
+						</Radio.Button>
+					</Radio.Group>
+
+					{/* {selectedView === VIEW_TYPES.OVERVIEW && <Overview host={host} />} */}
+					{/* {selectedView === VIEW_TYPES.METRICS && <Metrics host={host} />} */}
+				</>
```

Comment:
commented code

Reply:
@SagarRajput-7 This has been fixed in other PR.